internal/pfcpsim/server.go

```diff
@@ -22,6 +22,10 @@ import (
 // Its state is handled in internal/pfcpsim/state.go
 type pfcpSimService struct{}
 
+// SessionStep identifies the step in loops, used while creating/modifying/deleting sessions and rules IDs.
+// It should be high enough to avoid IDs overlap when creating sessions.
+const SessionStep = 10
```

Comment:
Does this step introduce any limitations in terms of apps, UEs etc.?

Reply:
Just saw the comment above. I believe 5 is enough. In theory with ROC limitations, we should expect max 8 applications (5 explicit applications + 3 filters to deny traffic to the RFC1918 IPs, in case we have a ALLOW-PUBLIC).

Reply:
Yep. Done